python/lsst/analysis/drp/gatherResourceUsage.py

```diff
@@ -331,6 +340,24 @@ def build_quantum_graph_cli(cls, argv):
             default=None,
             help="Data ID expression used when querying for input metadata datasets.",
         )
+        parser.add_argument(
+            "--output",
+            type=str,
+            help="Name of the output CHAINED collection.",
```

Comment:
I think this doc string should say that it will create the timestamp run (otherwise people have to read the output-run docs to work it out.)